<ui version="4.0" >
 <class>LoadPresetDialog</class>
 <widget class="QDialog" name="LoadPresetDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Load Preset</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDefault" >
      <attribute name="title" >
       <string>Default Presets</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="PresetListView" name="listDefault" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUser" >
      <attribute name="title" >
       <string>My Presets</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="PresetListView" name="listUser" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PresetListView</class>
   <extends>QListView</extends>
   <header>presetlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadPresetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadPresetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
